src/fsharp/service/ServiceInterfaceStubGenerator.fs

```diff
@@ -213,7 +213,7 @@ module internal InterfaceStubGenerator =
         let nm = 
             match arg.Name with 
             | None ->
-                if arg.Type.HasTypeDefinition && arg.Type.TypeDefinition.CompiledName = "Microsoft.FSharp.Core.Unit" then "()" 
+                if arg.Type.HasTypeDefinition && arg.Type.TypeDefinition.CompiledName = "unit" then "()" 
```

Comment:
Looks good, but can you check the namespace too, or something more?  Other people may define a type called "unit"